component/main.jsonnet

```diff
@@ -5,6 +5,71 @@ local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.keycloak_attribute_sync_controller;
 
-// Define outputs below
+local prefix = 'keycloak-attribute-sync-';
+
+local role = std.parseJson(kap.yaml_load('keycloak-attribute-sync-controller/manifests/controller/' + params.manifest_version + '/role.yaml'));
+local service_account = std.parseJson(kap.yaml_load('keycloak-attribute-sync-controller/manifests/controller/' + params.manifest_version + '/service_account.yaml'));
+local role_binding = std.parseJson(kap.yaml_load('keycloak-attribute-sync-controller/manifests/controller/' + params.manifest_version + '/role_binding.yaml'));
+local deployment = std.parseJson(kap.yaml_load('keycloak-attribute-sync-controller/manifests/controller/' + params.manifest_version + '/deployment.yaml'));
+
+local image = '%s/%s:%s' % [ params.images.controller.registry, params.images.controller.repository, params.images.controller.version ];
+
+local objects = [
+  role {
+    metadata+: {
+      name: prefix + super.name,
+    },
+  },
+  role_binding {
+    metadata+: {
+      name: prefix + super.name,
+    },
+    roleRef+: {
+      name: prefix + super.name,
+    },
+    subjects: std.map(function(s) s {
+      name: prefix + super.name,
+      namespace: params.namespace,
+    }, super.subjects),
+  },
+  service_account {
+    metadata+: {
+      name: prefix + super.name,
+    },
+  },
+  deployment {
+    metadata+: {
+      name: prefix + super.name,
+    },
+    spec+: {
+      template+: {
+        spec+: {
+          serviceAccountName: prefix + super.serviceAccountName,
+          containers: [
+            if c.name == 'manager' then
+              c {
+                image: image,
+                args: [],
```

Comment:
Why do we explicitly set `args: []` here?

Reply:
The default enables leader election which needs a million more objects (rbac rules/...) and patches until commodore supports kustomize...